py3xui/api/api.py

```diff
@@ -17,6 +17,9 @@ class Api:
         host (str): The XUI host URL.
         username (str): The XUI username.
         password (str): The XUI password.
+        token (str | None): The XUI secret token.
```

Comment:
Need more details in docstrings to explain how the new parameters work.
E.g. if you're using a self-signed certificate and will not change verify to False, it will...
Same for the AsyncApi so users can obtain all required info from the docstrings.

It's also recommended to add this information to the beginning of the docstring, not only yo description of arguments.

Reply:
Long story short: a user needs to understand how this works just on mouse hovering the object in IDE.